buildSrc/src/main/java/io/micronaut/guides/feature/MockServerClient.java

```diff
@@ -1,44 +0,0 @@
-package io.micronaut.guides.feature;
-
-import io.micronaut.core.annotation.NonNull;
-import io.micronaut.core.annotation.Nullable;
-import io.micronaut.starter.application.ApplicationType;
-import io.micronaut.starter.application.generator.GeneratorContext;
-import io.micronaut.starter.build.dependencies.Dependency;
-import io.micronaut.starter.feature.Feature;
-import io.micronaut.starter.feature.database.TestContainers;
-import jakarta.inject.Singleton;
-
-@Singleton
-public class MockServerClient implements Feature {
```

Comment:
was this not used?

Reply:
It collides with the one in v4.2.0 of Micronaut Starter

https://github.com/micronaut-projects/micronaut-starter/blob/4.2.x/starter-core/src/main/java/io/micronaut/starter/feature/test/MockServerClient.java

I assume this was added as a stop-gap until we got to use the one in 4.2.0?